src/widgets/settingspages/IgnoresPage.cpp

```diff
@@ -19,12 +20,14 @@
 #include <QTableView>
 #include <QVBoxLayout>
 
+namespace chatterino {
+
+using namespace literals;
+
 // clang-format off
-#define INFO "/block <user> in chat blocks a user.\n/unblock <user> in chat unblocks a user.\nYou can also click on a user to open the usercard."
+inline const QString INFO = u"/block <user> in chat blocks a user.\n/unblock <user> in chat unblocks a user.\nYou can also click on a user to open the usercard."_s;
```

Comment:
Can just inline this in `addUsersTab` instead, like you've done in HighlightingPage